src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -526,6 +531,26 @@ impl QemuInner {
             device
         ))
     }
+
+    fn hotplug_device(&mut self, device: DeviceType) -> Result<DeviceType> {
+        let qmp = match self.qmp {
```

Comment:
`hotplug_device` is based on the `self.qmp.is_some()` is ok. I think using the  `match` operation causes the None branch `dead_code`. Here just use unwrap() also can ensure its safety.

Reply:
Honestly I'm not sure about this one.  I'm totally fine with `unwrap()` if the check ensuring that it won't panic directly precedes it.  Here however it's in a different function so there are risks of this function being called from somewhere else in the future, or the current caller's code being modified without realising that a distant piece of code implicitly relied on its current behaviour.

It's true that the `None` branch cannot be reached currently, but it doesn't look that way to the compiler, or clippy for that matter.